notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -799,6 +799,49 @@ sacrifice for one's people [Admiral Adams blew himself up, taking out a number o
 stowaway [Noah stowed away on the interstellar ark in part to save his own skin, but mainly to be with his pregnant girlfriend, Hayley.]
 
 
+movie: Chaos Walking (2021)
+===========================
+
+:: Title
+Chaos Walking
+
+:: Description
+Chaos Walking is a 2021 American dystopian action film directed by Doug Liman
+from a screenplay by Patrick Ness and Christopher Ford. It is based on Ness's
+science fiction trilogy Chaos Walking, adapting its first book, 2008's The
+Knife of Never Letting Go.
+
+Synopsis: A young man lives in a dystopian world without women, where all
+living creatures can hear each other's thoughts in streams of images, words,
+and sounds called "Noise". When a woman crash-lands on the planet, he must
+help her escape danger.
+
+:: Date
+2021-02-24
+
+:: References
+https://en.wikipedia.org/wiki/Chaos_Walking_(film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+space colonization [The story is set in and around an all-male colony on the distant planet New World.]
+biased sex-ratio society [The story is explores life on an all-male space colony.]
+what if I heard all the thoughts around me [A main novelty of the film is that the New World males could hear each other's thoughts in streams of images, words, and sounds called "Noise".]
```

Comment:
https://www.themeontology.org/theme.php?name=telepathic%20society springs to mind...

Reply:
Good call. Added it.